src/pygeoprocessing/routing/routing.pyx

```diff
@@ -182,7 +182,7 @@ cdef int _is_close(double x, double y, double abs_delta, double rel_delta):
 # a class to allow fast random per-pixel access to a raster for both setting
 # and reading pixels.
 cdef class _ManagedRaster:
-    cdef LRUCache[int, double*]* lru_cache
+    cdef LRUCache[int, double*] * lru_cache

```

Comment:
for all the spaces that were added I'm surprised there's not one on the `double *` :)